### PR TITLE
Feature/weather app

### PR DESCRIPTION
**Pull request by widzthedvloper**
This pull request contains the following features:
- Just by entering a city name and clicking on the search button the user will be able to retrieve weather  data and get it to display on the screen
- the app uses a default city "San Francisco"  ;)
- the background change under 18 degree-Celcius, between 18 and 30 degree-Celcius, above 30 degree-Celcius

### CODE REVIEW

File: src/data.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { renderInfo, setAttribute, changeBackground } from './domManipulation';
+
+const apiInfo = (location) => {
```
Comment (addod19) [inline src/data.js:29]: Kindly use either Async and Await or Promise in getting the API info. For example; `const apiInfo = async (location) => {`

File: src/data.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      const temperature = Math.round((api.main.temp) - 273);
+      const city = api.name;
+      const { humidity } = api.main;
+      const minTemp = Math.round((api.main.temp_min) - 273);
+      const maxTemp = Math.round((api.main.temp_max) - 273);
+      const { description } = api.weather[0];
+      const { pressure } = api.main;
+
+      changeBackground(temperature);
+      // eslint-disable-next-line no-console
```
Comment (addod19) [inline src/data.js]: Kindly avoid inline disabling rules

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: REQUIRE CHANGES :recycle: :traffic_light: 

Hi @widzthedvloper, great job on this project so far :muscle: A user can search for a specific location :+1: The appearance is changed based on received data :100: However, let us kindly address the comments below:

- Kindly implement the toggle functionality, so we can toggle between the degree Celcius and Fahrenheit
- Kindly check the inline comments

[Optional]
- Kindly add a live link to the README file

Once these are resolved, kindly submit another code review request :smile:
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

File: src/data.js (hunk 1 above)
Comment (widzthedvloper) [reply src/data.js:29]: I did use Async and Await in getting the API info. The consumeData function is the one I use and it's the only function that I export from the data.js Module. And it's an asynchronous function.

**Review by mke2111 — APPROVED** [review]
# 😄 🥇  STATUS: APPROVED  🥇 👏🏽 

## Hey Happy Coder, 👋🏽 
Congratulations, 🥂 . You got minimum requirements for approval. 👍🏽 

### **_Well Done_** 💯 

    1. The toggle button switches the scale as expected
    2. The app runs just as expected and meets the requirements
    

### **_Room for Improvement_** 🤪 

    1. Add the name of all units being displayed on your app's page. Also, add the SI units for each quantity. See the image below
Screenshot from 2021-06-03 02-50-38

### You may now merge to the develop branch and remember to submit a project completion form.
### I wish you the very best in your upcoming projects.

**Cheers 🍹 and Happy coding 💻** 

<hr>

### Best regards
Slack 🆘  Roy Mukuye
Twitter 🐦  [Roy Mukuye](https://twitter.com/Roymkenya)
LinkedIn 🔗  [Roy Mukuye](https://www.linkedin.com/in/roy-mukuye/)

**Pull request merged**